### PR TITLE
remove the concepts of 'admission', 'admitting'

### PR DESCRIPTION
**Pull request by mchalski**
https://tracker.mender.io/browse/MEN-2223

basic terminology:
- `admit` goes out; the user `authorizes/accepts` the device; the device `authenticates` with the server
- so e.g. there are `two authorization workflows - preauth and accept-on-request`

more advanced changes:
- deviceauth api bumped to v2
- removed lots of extra explanations of devauth vs devadm url conventions - obsolete
- tutorial sections are simplified because of a better v2 api, some troubleshooting unnecessary/reworded
- chaged urls and comments also in diagrams
- random, obvious fixes

### CODE REVIEW

File: 06.Server-integration/02.Preauthorizing-devices/docs.md [hunk 1]
```diff
@@ -116,30 +116,30 @@ In the event that the decommissioning operation fails, perform a [manual databas
...
+!!! Adjust the variable value to the actual identity of your device. If you have several identity attributes in your identity scheme, separate them with commas in JSON format inside this single object, for example `DEVICE_IDENTITY_JSON_OBJECT_STRING='{"mac":"02:12:61:13:6c:42", "serialnumber":"1928819"}'`.
 
-Secondly, set the contents of the device public key you generated above in a second variable (with newlines written out literally):
+Secondly, set the contents of the device public key you generated above in a second variable:
 
 ```bash
-DEVICE_PUBLIC_KEY="$(cat keys-client-generated/public.key | sed -e :a  -e 'N;s/\n/\\n/;ta')\n"
+DEVICE_PUBLIC_KEY="$(cat keys-client-generated/public.key)"
```
Comment (estenberg) [inline 06.Server-integration/02.Preauthorizing-devices/docs.md]: Will this work now? I thought we still had to do the "sed trick" due to how the shells work?

Comment (mchalski) [reply 06.Server-integration/02.Preauthorizing-devices/docs.md]: oh, could be.  today I'll be exercisig all the commands, urls, etc. i'll followup with fixes if necessary.

Comment (mchalski) [reply 06.Server-integration/02.Preauthorizing-devices/docs.md]: @estenberg 

you were right - the sed stays. I left in are some minor simplifications, verified they work.

one important change: json fields for v2 are also updated (key -> pubkey, device_identity -> identity_data).

I also verified all urls, images, examples render correctly, so we're good to merge.

Comment (estenberg) [reply 06.Server-integration/02.Preauthorizing-devices/docs.md]: :+1:

**Pull request merged**